scripts/gulp/copy.js

```diff
@@ -15,7 +15,11 @@ const baseManifest = JSON.parse( fs.readFileSync( './package.json', 'utf8' ) );
  * @returns {Object} An output stream from gulp.
  */
 function copyComponentsBoilerplate() {
-  return gulp.src( [ './src/' + ( parseComponentName || '*' ), '!./src/*.js', '!./src/*.less' ] )
+  return gulp.src( [
+    './src/' + ( parseComponentName || '*' ),
+    '!./src/*.js',
+    '!./src/*.less'
+  ] )
```

Comment:
Should L:19-22 be indented 2 more spaces to align with L:23?

Reply:
I don't think so. Closing brackets/parens appear on the same indent level as where they are opened. Otherwise we'd be indenting L:29 too, which I don't think would make sense.

Reply:
Err I see that needs to be fixed in the lint.js